### DISCUSSION TITLE
Instead of just stretching grey background - stretch horizontally and split full chat in many splits?

### DISCUSSION THREAD

**Discussion by gorro123** [Chatterino/chatterino2#3533] [Ideas]
I used the search function and didn't find anything.

If you open like 5 windows of the same chat, put all those windows next to each other and scroll to different spots of the chat you use more of your typical 16:9 screen to see more of the chat.

In Chatterino you can zoom out to see more of the chat which(unreadable), but if you stretch Chatterino horizontally it doesnt split into another split so you can actualy see more of the chat.
Would it be possible to add that?
Add a setting: Stretch chat horizontally into many splits on/off

I hope you know what I mean.

**Comment by pajlada** [comment]
I don't understand - could you provide some pictures of what you would like to achieve?

**Comment by gorro123** [comment]
![example](https://user-images.githubusercontent.com/38992318/150683105-adf220c5-40b5-464a-9937-6c24fa6a4f6d.PNG)
In Chatterino autoscrolling is disabled as soon as you scroll, so I can't record of what I am talking about.
But the goal is to have it like in this picture: 5 times the same chat side by side without any gap in between like in that example.
And achieve that by simply stretching 1 chat window.
@pajlada

**Reply by pajlada** [reply]
Alright I understand, however this would require a pretty major overhaul of our rendering system for a niche feature - if you or someone else manage to figure this feature out I'd be happy to review a PR if it's not too disruptive to the codebase, but it's not something I think I will spend time to focus on myself.
